savaslabs/cyrano
# Update dependencies to avoid warnings

This PR adds an upgrade for these two dependencies:

- `react-native@0.70.14`
- `react-native-svg@13.4.0`

To test:

- Just run `yarn install`.
- Run `npx expo start -c`
- It should work with `node 16`
